plugins/filter_cgolib/Readme.md

```diff
@@ -0,0 +1,111 @@
+
+
+call golib from c
+
+example go src:
+
+```go
+
+package main
+
+import "C"
+import (
+	"fmt"
+)
+
+//export GoLibInit
+func GoLibInit(name, value []string) int {
+
+	fmt.Printf("get name %#v. ", name)
+	fmt.Printf("get value %#v. ", value)
+
+	return 0
```

Comment:
It would be nice to replace this with a constant like `FLB_OK`. I get this from the `fluent-bit-go` library.